### DISCUSSION TITLE
Include filepath in search criteria?

### DISCUSSION THREAD

**Discussion by natdm** [ibhagwan/fzf-lua#1515] [Q&A] [answered]
I'm migrating from fzf-vim and one of my more common searches I did was to search for a filepath, as well as search for some code. eg: 

`someFunction packages-foo | packages-bar`

How can I enable this on one of the search settings (preferably project)?

**Comment by ibhagwan** [comment, marked as answer]
grep_project excludes file path for convenience, use grep with empty search string and you’ll be able to search the paths:
```lua
:lua require'fzf-lua'.grep({ search = '' })
```

**Comment by natdm** [comment]
Thank you!
